README.md

```diff
@@ -145,16 +174,17 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
+Command: g++ -std=c++14 sentence.cpp -o sentence && ./sentence
```

Comment:
Thank you, thank you, for restoring my faith in C++. This is one of the few C++ submissions that didn't kill off a small corner of my soul.

# +3 bonus

You had good tests, both positive and negative, and you used a table to control the state machine.